package.json

```diff
@@ -6,10 +6,10 @@
   "description": "GitHub Desktop build dependencies",
   "scripts": {
     "cli": "ts-node --require ./app/test/globals.ts --require ./app/src/cli/dev-commands-global.ts app/src/cli/main.ts",
-    "test:integration": "cross-env TEST_ENV=1 ELECTRON_NO_ATTACH_CONSOLE=1 xvfb-maybe --auto-servernum -- mocha -t 30000 --require ts-node/register app/test/integration/*.ts",
-    "test:unit": "cross-env ELECTRON_RUN_AS_NODE=1 node_modules/.bin/electron ./node_modules/jest/bin/jest --detectOpenHandles --silent --config ./jest.config.js",
+    "test:integration": "cross-env TEST_ENV=1 ELECTRON_NO_ATTACH_CONSOLE=1 xvfb-maybe --auto-servernum -- mocha -t 30000 --require ts-node/register app/test/integration/*.ts --reporter mocha-multi-reporters --reporter-options configFile=./app/test/mochareporters.json",
+    "test:unit": "cross-env ELECTRON_RUN_AS_NODE=1 node_modules/.bin/electron ./node_modules/jest/bin/jest --detectOpenHandles --silent --config ./jest.config.js --reporters=default --reporters=jest-junit",
```

Comment:
could `--reporters=jest-junit` be moved into `jest.config.js` too? or is the idea to override the `cobertura` option supplied there?

Reply:
I think it’s like this so the JUnit file is only generated once.

Reply:
@outofambit We can move this option to` jest.config.js`. It will be like  `reporters: ['default', 'jest-junit']`
`cobertura` is a different option that is included in `coverageReporters: ['text-summary', 'json', 'html', 'cobertura']`

Reply:
@outofambit 
Shall I make this change?

Reply:
@vagisha-nidhi yes if you could make that change that would be great. thank you!

Reply:
@outofambit I have added reporter to jest config. Please have a look.